.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,70 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
```

Comment:
you can change this however you want, seems docker likes to do things with `v*` as a standard.